ddagent.py

```diff
@@ -398,6 +398,8 @@ def __init__(self, port, agentConfig, watchdog=True,
         self._metrics = {}
         AgentTransaction.set_application(self)
         AgentTransaction.set_endpoints(agentConfig['endpoints'])
+        if agentConfig['endpoints'] == {}:
```

Comment:
it'd be good to check the case when there is an endpoint but no attached api_keys and log the same warning when that happens

(otherwise in that case the forwarder drops all incoming payloads but doesn't really explain why)